From 46c158358915692c7fe95db9f1fc75e9a376d459 Mon Sep 17 00:00:00 2001
From: Christophe Priouzeau <christophe.priouzeau@linaro.org>
Date: Tue, 9 Jul 2019 14:03:04 +0200
Subject: [PATCH] optee-test: fix build failure with GCC 6

Add -Wno-unused-const-variable to disable errors introduced with the upgrade
to GCC 6:
error: 'pbkdf2_4_dkm' defined but not used [-Werror=unused-const-variable=]

Signed-off-by: Andrey Konovalov <andrey.konovalov@linaro.org>
---
 host/xtest/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/host/xtest/Makefile b/host/xtest/Makefile
index d0f629b..a32bfd6 100644
--- a/host/xtest/Makefile
+++ b/host/xtest/Makefile
@@ -165,7 +165,8 @@ CFLAGS += -Wall -Wcast-align -Werror \
 	  -Wwrite-strings \
 	  -Wno-declaration-after-statement \
 	  -Wno-unsafe-loop-optimizations \
-	  -Wno-missing-field-initializers -Wno-format-zero-length
+	  -Wno-missing-field-initializers -Wno-format-zero-length \
+	  -Wno-unused-const-variable
 endif
 
 CFLAGS += -g3
-- 
2.17.1

